### ISSUE TITLE
[Fabric] `apugli:modify_scale` not being synced to client properly

### ISSUE THREAD

**Issue by CyraLX** [MerchantPug/apugli#60]
With the example power regardless of what `delay` is set to when the player respawns (and maybe more similar means) their scale gets reset to defaults on the client but on the server it remains the same (modified scale).
This create an issue of desync that the player appears to be human sized but in reality is bigger and creating obvious issues.

Example Power:
```json
{
	"type": "apugli:modify_scale",
	"delay": 0,
	"scale_types": [
		"pehkui:visibility",
		"pehkui:width",
		"pehkui:height",
		"pehkui:drops"
	],
	"modifier": {
		"operation": "multiply_base_additive",
		"value": 0.3
	}
}
```

**Comment by ChrysanthCow** [comment]
I can't seem to reproduce this issue, please check your logs and see if there's anything out of the ordinary (besides a single error message from a packet that I've now fixed).

**Comment by ChrysanthCow** [comment]
The rewrites have probably fixed this, I'll probably leave this open until I get confirmation.

**Comment by CyraLX** [comment]
This is has been fixed with 2.8.1 yes, I forgot to comment to confirm it
